ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/address-locking.j2

```diff
@@ -0,0 +1,49 @@
+{# address locking #}
+{% if address_locking is arista.avd.defined %}
+
+# Address Locking
+
+## Address Locking Summary
+
+| Settings | Value |
+| -------- | ----- |
+| Disable IP locking on configured ports | {{ address_locking.disabled | arista.avd.default(false) }} |
+{%     if address_locking.local_interface is arista.avd.defined %}
+| Local Interface | {{ address_locking.local_interface | arista.avd.default("-") }} |
+{%     endif %}
+{%     if address_locking.locked_address.expiration_mac_disabled is arista.avd.defined(true) %}
+| Disable deauthorizing locked addresses upon MAC aging out | True |
+{%     endif %}
+{%     if address_locking.locked_address.ipv4_enforcement_disabled is arista.avd.defined(true) %}
+| Disable enforcement for locked ipv4 addresses | True |
+{%     endif %}
+{%     if address_locking.locked_address.ipv6_enforcement_disabled is arista.avd.defined(true) %}
+| Disable enforcement for locked ipv6 addresses | True |
+{%     endif %}
+{%     if address_locking.dhcp_servers_ipv4 is arista.avd.defined %}
```

Comment:
Maybe let's make these bullet points rather than a table like we do in other templates?

if not then at `Settings` -> `Setting` for the header

Reply:
ok so whichever is more appropriate here - in any case let's fix the header if we keep it like this